woocommerce/woocommerce-gutenberg-products-block
# Update Storybook and alias `wordpress-components`

I'm not 100% sure why it happens (possibly due to how we map to remote/external?) but in order to build Storybook without warnings we need to alias the `wordpress-components` package. Without this it gives warnings about `useSlot` being missing.

Fixes #3060

### How to test the changes in this Pull Request:

1. `npm run storybook`
2. Confirm no warnings in terminal
3. http://localhost:6006/ runs correcrly.
